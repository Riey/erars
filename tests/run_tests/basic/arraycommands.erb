@SYSTEM_TITLE
#DIM TEST_VARS = 1,2,3,4,5,6
ARRAYREMOVE TEST_VARS, 2, 3
REPEAT 6
    PRINTV TEST_VARS:COUNT
REND
PRINTL

TEST_VARS:2 = 3
TEST_VARS:3 = 4
TEST_VARS:4 = 5
TEST_VARS:5 = 6

ARRAYSORT TEST_VARS
REPEAT 6
    PRINTV TEST_VARS:COUNT
REND
PRINTL

ARRAYSORT TEST_VARS, BACK, 0, 4
REPEAT 6
    PRINTV TEST_VARS:COUNT
REND
PRINTL

ARRAYSORT TEST_VARS, FORWARD, 0, 4
REPEAT 6
    PRINTV TEST_VARS:COUNT
REND
PRINTL
