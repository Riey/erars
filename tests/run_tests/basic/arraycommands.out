126000
123456
432156
123456
1234560000
